csaf_2.1/prose/edit/src/schema-elements-01-defs-03-full-product-name.md

```diff
@@ -252,8 +253,11 @@ The package URL (purl) representation (`purl`) is a `string` of 7 or more charac
 > CSAF uses only the canonical form of purl to conform with section 3.3 of [cite](#RFC3986).
 > Therefore, URLs starting with `pkg://` are considered invalid.
 
-This package URL (purl) attribute refers to a method for reliably identifying and locating software packages external to this specification.
-See [cite](#PURL) for details.
+A package URL (purl) attribute refers to a method for reliably identifying and locating software packages external to this specification.
+See [cite](#PURL) for details. Multiple purls can be specified to allow for identifiers to locate identical components in different locations.
+
+If multiple purls are specified, they MUST only differ in their qualifiers. Otherwise, separate product branches should
+be used to differentiate between the components.
```

Comment:
```suggestion
If multiple purls are specified, they SHALL only differ in their qualifiers. 
Otherwise, separate product branches should be used to differentiate between the components.